tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/group_275_event_275.pysnap

```diff
@@ -13,7 +13,7 @@ app-depth-1:
           frame*
             function*
               "stripped_application_code"
-        type (ignored because exception is synthetic (detected via exception type))
```

Comment:
the fact that this extra hint is gone is a bit unfortunate but I think acceptable

Reply:
We could restore it by adding another flag `synthetic_detected` to the `Mechanism` interface. This would also allow us to restore the old behavior of only skipping the system type component if the synthetic property was detected.